src/app/launcher/create-app/project-summary-createapp-step/project-summary-createapp-step.component.html

```diff
@@ -165,8 +165,8 @@ <h2 class="card-pf-title">
             </div>
             <div class="card-pf-body">
               <div class="list-group-item"
-                   [ngClass]="{'suggested': summary?.pipeline?.suggested === true || summary?.pipeline?.techpreview === true}">
-                <div class="group" [ngClass]="{'with-tag': summary?.pipeline?.suggested === true || summary?.pipeline.techpreview === true}">
+                   [ngClass]="{'suggested': summary?.pipeline?.suggested === true || summary?.pipeline?.techPreview === true}">
```

Comment:
Could you remove the `=== true` when possible. I know it's there a bit everywhere, but it's easier to read without :p

Reply:
Right, that was a quick search/replace. Should do more boyscoutting. Consider it done :)

Reply:
6f67038